examples/63_Conic.html

```diff
@@ -36,12 +37,45 @@
 		              {name:"Co", type:"ConicBy5", args:["A","B","C","D","E"],color:[0,0,1],alpha:1,size:1},
 //		     {name:"l", type:"Join", args:["A", "D"]}
                       ];
-            createCindy({canvasname:"CSCanvas",
+            var cdy=createCindy({canvasname:"CSCanvas",
                         movescript:"csmove",
                         grid:1,
                         snap:true,
                         geometry:gslp});
 
+            var cases = [
+                ["parabola1", [0,0,1], [1,1,1], [2,4,1], [-2,4,1], [-1,1,1]],
```

Comment:
Also something strange I noticed (perhaps something inadvertently was changed by me), clicking the buttons 'circle1',  'parabola2' and 'parabola1' in that order ends up drawing only the left portion of the 'parabola1'.